src/mfa.js

```diff
@@ -0,0 +1,108 @@
+import axios from "axios";
+import { setCookiesAndTokens } from "./cookies.js";
+import { store } from "./store.js";
+import { getQueryAttr, redirectToPath } from "./url.js";
+import { exchange } from "./refresh.js";
+import { throwFormattedError } from "./utils.js";
+
+/**
+ * Send an SMS to a phone number
+ * @param {String} type Type of SMS to send
+ * @param {String} to Phone number in E.164 format
+ * @param {String} firstFactorCode Identifier obtained from login() response
+ * @returns {Object}
+ */
+export async function sendSms({ type, to, firstFactorCode } = {}) {
+  if (!type) {
+    throw new Error('Userfront.sendSms called without "type" property.');
+  }
+
+  switch (type) {
+    case "securityCode":
```

Comment:
This is really to just leave the function open-ended for `link` (and any other `type`s we may implement).

Reply:
We refer to this as `strategy` everywhere else. Did you intentionally use `type` here with that in mind, or should we use `strategy` here as well?